HaulageApplication/Haulage/DatabaseSetup.cs

```diff
@@ -34,9 +34,9 @@ public static void InitializeDatabase()
         {
             using (var connection = new SQLiteConnection(GetDatabasePath()))
             {
-                DropTables(connection);
-                CreateTables(connection);   
-                GenerateData(connection);
+                //DropTables(connection);
+                //CreateTables(connection);   
+                //GenerateData(connection);
```

Comment:
why are these commented out, this makes the InitializeDatabase method effectively do nothing? If you don't want those things to happen in your test (in DatabaseTests.cs) then just don't called InitializeDatabase there, rather than making it do nothing.

Reply:
uncommented